Project.toml

```diff
@@ -0,0 +1,36 @@
+name = "PowerModels"
+uuid = "c36e90e8-916a-50a6-bd94-075b64ef4655"
+authors = ["Carleton Coffrin"]
+repo = "https://github.com/lanl-ansi/PowerModels.jl"
+version = "0.9.6"
+
+[deps]
+InfrastructureModels = "2030c09a-7f63-5d83-885d-db604e0e9cc0"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+
+[compat]
+Cbc = ">= 0.4"
+InfrastructureModels = "< 0.2"
+Ipopt = ">= 0.4"
+JuMP = "< 0.19"
+Juniper = "< 0.4"
+Memento = ">= 0.8, < 0.11"
+Pavito = "^0.1"
+SCS = ">= 0.4"
+julia = "^1"
```

Comment:
Remember that packages only in `extras` will cause an error if they are in `compat` (for now, I believe the Registrator needs a new release before we can add those to `compat`, which will hopefully happen soon)

Reply:
Good point.  Let's wait and see if that happens in the next few days before releasing.

Reply:
You can handle the version requirements in the Extras using the Manifest.TOML temporarily.

Reply:
I'm too lazy to maintain that at the moment.  If the registator does not get fixed soon, we can just drop these for now.